README.md

```diff
@@ -185,6 +185,26 @@ You can set the default region through the `defaultRegion` property:
     private $phoneNumber;
 ```
 
+You can get the region dynamically through the `regionProperty` property:
```

Comment:
Could you add a bit more information to explain what you can achieve with this new property please?
The goal is obviously to change the default region according to the user but I think we could explain it. :)